exercises/calculator.cpp

```diff
@@ -10,3 +10,40 @@
   Multiplication: 8
   Division: 2
 */
+
+
+#include <iostream>
+using namespace std;
+
+int main() {
+    double num1, num2;
+
+    // Input dei numeri
+    cout << "Insert first number: ";
+    cin >> num1;
+    cout << "Insert second number: ";
+    cin >> num2;
+
+    // Operazioni
+    double sum = num1 + num2;
+    double difference = num1 - num2;
+    double multiplication = num1 * num2;
+    double division;
+
+    // Controllo divisione per zero
+    if(num2 != 0) {
+        division = num1 / num2;
+    } else {
+        cout << "Division by zero is not allowed." << endl;
+    }
+
+    // Output
+    cout << "SUM: " << sum << endl;
+    cout << "Difference: " << difference << endl;
+    cout << "Multiplication: " << multiplication << endl;
+    if(num2 != 0) {
+        cout << "Division: " << division << endl;
+    }
```

Comment:
Questo controllo già lo fai a riga 34. Potresti anche inserire un'istruzione di chiusura di programma con errore oppure eventualmente chiedere di nuovo un numero in input all'utente.

```cpp
#define EXIT_FAILURE -1;

// code
if(){
    ...
}
else {
    return EXIT_FAILURE;
    }

```